website/docs/terms/dataframe.md

```diff
@@ -0,0 +1,24 @@
+---
+id: dataframe
+title: Data Frame
+displayText: dataframe  
+hoverSnippet: A data frame is a two-dimensional data structure (rows and columns). It's the most common way of representing and interacting with large datasets in Python.
+---
+:::important This page could use some love
+This term would benefit from additional depth and examples. Have knowledge to contribute? [Create a discussion in the docs.getdbt.com GitHub repository](https://github.com/dbt-labs/docs.getdbt.com/discussions) to begin the process of becoming a glossary contributor!
+:::
+
+<!--- Useful reference: https://databricks.com/glossary/what-are-dataframes --->
+
+A data frame is a way of storing and manipulating data in Python. (It's also used in other languages popular for data processing, such as R and Scala.)
```

Comment:
Would it make sense to do "manipulating data in Python" => "manipulating **tabular** data in Python"